test/inductor/test_torchinductor.py

```diff
@@ -767,16 +767,16 @@ def fn(a):
         self.common(fn, ((torch.rand((10, 3, 352, 352), dtype=torch.float16),)))
 
     def test_expanded_reduction(self):
-        if self.device == "cpu":
-            raise unittest.SkipTest(
-                "https://github.com/pytorch/torchdynamo/issues/1697"
-            )
-
         def fn(x, y):
             z = x * y
             return z.sum((0, 1))
 
-        self.common(fn, (torch.randn(2, 197, 256), torch.randn(2, 1, 256)))
+        # Mismatched elements: 1 / 256 (0.4%)
+        # Greatest absolute difference: 2.002716064453125e-05 at index (36,) (up to 1e-05 allowed)
+        # Greatest relative difference: 4.907013266928203e-06 at index (36,) (up to 1.3e-06 allowed)
```

Comment:
Only adding the comments of this error message seems confusing. Maybe explaining why we are using looser thresholds here is more clear?